README.md

```diff
@@ -1,31 +1,36 @@
-# Open Container Distributable Image Specification
+# Open Container Initiative Image Format Specification
 
-The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
+The OCI Image Format project creates and maintains the software shipping container image format spec (OCI Image Format). The goal of this specification is to create interoperable tools for building, transporting, and preparing a container image to run.
```

Comment:
s/...is to create interoperable.../...is to enable the creation of interoperable.../